src/Upgrade/Domain/Strategy/ReleaseApp/Validator/Threshold/PathThresholdValidator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrade\Domain\Strategy\ReleaseApp\Validator\Threshold;
+
+use ReleaseAppClient\Domain\Dto\Collection\ReleaseGroupDtoCollection;
+use Upgrade\Infrastructure\Configuration\ConfigurationProvider;
+use Upgrade\Infrastructure\Exception\UpgraderException;
+
+class PathThresholdValidator implements ThresholdValidatorInterface
+{
+    /**
+     * @var \Upgrade\Infrastructure\Configuration\ConfigurationProvider
+     */
+    protected ConfigurationProvider $configurationProvider;
+
+    /**
+     * @param \Upgrade\Infrastructure\Configuration\ConfigurationProvider $configurationProvider
+     */
+    public function __construct(ConfigurationProvider $configurationProvider)
+    {
+        $this->configurationProvider = $configurationProvider;
+    }
+
+    /**
+     * @param \ReleaseAppClient\Domain\Dto\Collection\ReleaseGroupDtoCollection $releaseGroupDtoCollection
+     *
+     * @throws \Upgrade\Infrastructure\Exception\UpgraderException
+     *
+     * @return void
+     */
+    public function validate(ReleaseGroupDtoCollection $releaseGroupDtoCollection): void
+    {
+        $softThreshold = $this->configurationProvider->getSoftThresholdBugfix();
+        if ($releaseGroupDtoCollection->getCommonModuleCollection()->getPathAmount() > $softThreshold) {
+            throw new UpgraderException(
+                sprintf('Soft threshold hit by %s major releases amount', $softThreshold),
```

Comment:
amount word is not needed, just releases suffices. amount is a specific value not a count btw.

Reply:
Removed "amount". Thanks a lot!